lib/terjira/option_supportable.rb

```diff
@@ -30,8 +30,9 @@ def self.included(klass)
     # Transforming and clening options
     # and suggest list of option values
     def suggest_options(opts = {})
+      self.with_editor = options[:editor]
       origin = options.dup
-      self.with_editor = opts[:editor]
+      origin.delete(:editor)
```

Comment:
This was causing the issues earlier. Other logic that parses the options in the client couldn't handle the that the `editor` option was in the hash.